src/module/rules/rule-element/item-alteration/alteration.ts

```diff
@@ -374,6 +378,64 @@ class ItemAlteration extends foundry.abstract.DataModel<RuleElementPF2e, ItemAlt
                 }
                 return;
             }
+            case "potency": {
+                if (game.pf2e.variantRules.AutomaticBonusProgression.isEnabled(this.actor)) {
+                    return;
+                }
```

Comment:
Define an `abpEnabled` further up top and reuse that so we're not eating kilobytes of whitespace